extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -198,15 +198,19 @@ export class ProjectsController {
 		}
 
 		const sqlProjectsService = await utils.getSqlProjectsService();
-		if (utils.getAzdataApi()) {
-			const projectStyle = creationParams.sdkStyle ? mssql.ProjectType.SdkStyle : mssql.ProjectType.LegacyStyle;
-			await (sqlProjectsService as mssql.ISqlProjectsService).createProject(newProjFilePath, projectStyle, targetPlatform);
-		} else {
-			throw new Error(constants.errorNotSupportedInVsCode('createProject'));
-			//const projectStyle = creationParams.sdkStyle ? mssqlVscode.ProjectType.SdkStyle : mssqlVscode.ProjectType.LegacyStyle;
-			//await (sqlProjectsService as mssqlVscode.ISqlProjectsService).createProject(newProjFilePath, projectStyle, targetPlatform);
+		try {
+			if (utils.getAzdataApi()) {
+				const projectStyle = creationParams.sdkStyle ? mssql.ProjectType.SdkStyle : mssql.ProjectType.LegacyStyle;
+				await (sqlProjectsService as mssql.ISqlProjectsService).createProject(newProjFilePath, projectStyle, targetPlatform);
+			} else {
+				const projectStyle = creationParams.sdkStyle ? mssqlVscode.ProjectType.SdkStyle : mssqlVscode.ProjectType.LegacyStyle;
+				await (sqlProjectsService as mssqlVscode.ISqlProjectsService).createProject(newProjFilePath, projectStyle, targetPlatform);
+			}
+		} catch (e) {
+			throw (console.log);
```

Comment:
is this try-catch leftover from debugging?  Also, it's just throwing a function pointer (not invoking the `console.log()` function).

The way we've been doing error handling with the new Projects library is to call `throwIfFailed()` in projects.ts.  Maybe we could push this logic into a static function in the `Project` class, and convert `throwIfFailed()` to a static call, that way this file doesn't have to know or care whether it's in the ADS or VS Code context.

Reply:
Yes, leftover from testing. Removed it. 
I can create a workitem to add "throwIfFailed" to keep the scope of this PR small. Thoughts?